e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/assertions/AssertionEMTest.kt

```diff
@@ -0,0 +1,136 @@
+package org.evomaster.e2etests.spring.openapi.v3.assertions
+
+import com.foo.rest.examples.spring.openapi.v3.assertions.AssertionController
+import org.evomaster.client.java.instrumentation.shared.ClassName
+import org.evomaster.core.problem.rest.HttpVerb
+import org.evomaster.e2etests.spring.openapi.v3.SpringTestBase
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertTimeoutPreemptively
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.Duration
+
+/**
+ *
+ */
+
+class AssertionEMTest : SpringTestBase() {
+
+    companion object {
+        @BeforeAll
+        @JvmStatic
+        fun init() {
+            initClass(AssertionController())
+        }
+    }
+
+
+
+    @Test
+    fun testRunEMAssertionsOff() {
+        val outputFolderName = "AssertionEM"
+        val className = ClassName("org.foo.AssertionEMOff")
+
+        testRunEMGeneric(false, className)
+
+        val assertion = Files.lines(Paths.get(outputFolderPath(outputFolderName)
+                + "/"
+                + className.getBytecodeName()
+                + ".kt")).noneMatch { l: String -> l.contains(".assertThat()") }
+
```

Comment:
this code is copied&pasted several times. should be refactored in a common function in the superclass, or test utility class

Reply:
I've included it as a separate function in the current test class. Would it be used enough to require a separate test utility class?